core/lib/refinery.rb

```diff
@@ -3,6 +3,8 @@
 require 'action_controller'
 require 'rbconfig'
 require 'font-awesome-sass'
+require 'jquery-rails'
+require 'jquery-ui-rails'
```

Comment:
It's a dependency of core : https://github.com/refinery/refinerycms/blob/dc6aa979600405613f0d5a181e92cf3eb4f0e657/core/refinerycms-core.gemspec#L35

I will check if we still need to require it here. A long time ago, it fixed a bug.